opencti-platform/opencti-front/src/private/components/profile/ProfileLocalStorage.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import Paper from '@mui/material/Paper';
+import makeStyles from '@mui/styles/makeStyles';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogContent from '@mui/material/DialogContent';
+import Dialog from '@mui/material/Dialog';
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+
+import DialogActions from '@mui/material/DialogActions';
+import { useFormatter } from '../../../components/i18n';
+import Transition from '../../../components/Transition';
+
+const useStyles = makeStyles(() => ({
+  paper: {
+    width: '100%',
+    margin: '0 auto',
+    marginBottom: 30,
+    padding: 20,
+    textAlign: 'left',
+    borderRadius: 4,
+    position: 'relative',
+  },
+}));
+
+const ProfileLocalStorage: React.FC = () => {
+  const { t_i18n } = useFormatter();
+  const classes = useStyles();
+
+  const [displayConfirmation, setDisplayConfirmation] = useState(false);
+
+  return (
+    <>
+      <Paper
+        classes={{ root: classes.paper }}
+        variant="outlined"
+        sx={{ height: 'unset' }}
+      >
+        <Typography
+          variant="h1"
+          gutterBottom={true}
+        >
+          {t_i18n('Local settings')}
+        </Typography>
+        <pre>
+          {
+            t_i18n('Your local storage contains the latest filters and searches used in most views of the platform. Clearing local storage might help to load a page crashing because of some filtering, searching or sorting issue.')
+          }
+        </pre>
+        <Button
+          variant="contained"
+          color="primary"
+          onClick={() => setDisplayConfirmation(true)}
+        >
+          {t_i18n('Clear local storage')}
+        </Button>
+      </Paper>
+
+      {/* Confirmation dialog */}
+      <Dialog
+        open={displayConfirmation}
+        PaperProps={{ elevation: 1 }}
+        keepMounted={false}
+        TransitionComponent={Transition}
+        onClose={() => setDisplayConfirmation(false)}
+      >
+        <DialogTitle>
+          {t_i18n('Clear local storage')}
+        </DialogTitle>
+        <DialogContent>
+          <Alert
+            icon={false}
+            severity="warning"
+            variant="outlined"
+            sx={{ color: 'text.primary' }}
+          >
+            <AlertTitle style={{ marginBottom: 0, fontWeight: 400 }}>
+              {t_i18n('This will erase all the views settings you have made. All these changes will be lost. Are you sure?')}
```

Comment:
Not a blocker but
```suggestion
              {t_i18n('This will erase all the views settings that you have made. All these changes will be lost. Are you sure?')}
```
I thing it's either "you made" or "that you have made"